README.rst

```diff
@@ -49,6 +49,10 @@ environment by running::
     docker compose up -d
 
 
+Then, create the application's data layer:
```

Comment:
reStructuredText needs two colons to have a literal code block created.

```suggestion
Then, create the application's data layer::
```

Reply:
resolved in f2a927a